frontend/templates/layouts/example-pages/amp-ads-page.j2

```diff
@@ -3,7 +3,7 @@
 <head>
   <meta charset="utf-8">
 {% if not example.has_tag_in_head('title') %}
-  <title>{{export.title|e}} - {{_('Example')}} {{example.index}}</title>
+  <title>{{export.title|e}} - {{_('Example')|e}} {{example.index|e}}</title>
```

Comment:
```suggestion
  <title>{{export.title|e}} - {{_('Example')}} {{example.index}}</title>
```

It's safe to assume that the result of `_('Example')` will not contain any HMTL chars. The same for `example.index`.